www/package.json

```diff
@@ -48,12 +48,21 @@
         ]
     },
     "devDependencies": {
+        "@types/enzyme": "^3.10.8",
         "@types/react-router-dom": "^5.1.7",
         "@typescript-eslint/eslint-plugin": "^4.25.0",
         "@typescript-eslint/parser": "^4.25.0",
+        "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
+        "enzyme": "^3.11.0",
+        "enzyme-to-json": "^3.6.2",
         "eslint": "^7.27.0",
         "eslint-plugin-prettier": "^3.4.0",
         "eslint-plugin-react": "^7.23.2",
         "prettier": "^2.3.0"
+    },
+    "jest": {
```

Comment:
Should this live in a `jest.config.js` file. This is the pattern we are following on the backend I believe.

Reply:
I would like to but I think to make jest work for react, `create-react-app` has a preconfigured template and their guidance to override it is to put it in `package.json`. I tried exporting their default config but still no luck and am getting the following errors
![image](https://user-images.githubusercontent.com/43416725/121832324-07e48480-cc87-11eb-9140-57acbea5244a.png)

Reply:
Okay that is fine then.